.github/workflows/scripts/get-codeowners.sh

```diff
@@ -9,6 +9,10 @@
 
 set -euo pipefail
 
+get_codeowners() {
+  echo "$((grep -m 1 "${1}" .github/CODEOWNERS || true) | sed 's/   */ /g' | cut -f3- -d ' ')"
```

Comment:
This is the exact same logic as before, just using the argument passed to the function instead of a global variable.